### DISCUSSION TITLE
Always on top

### DISCUSSION THREAD

**Discussion by aserwen** [baffler/Transparent-Twitch-Chat-Overlay#90] [Q&A]
Is there a way to activate an always on top, so it will keep showing when i open a game or a full window program?

**Comment by Fyr01** [comment]
Hey hello!

 I have the same Question &/or request
 
 is there a way to put it above the full game screens?
To be able to continue seeing the chat overlay on the game screen mode and be able to read the chat? 
Thanks in advance.

**Comment by GargosTheMighty** [comment]
Use borderless window mode for your game.
